server/src/main/java/org/opentosca/toscana/plugins/kubernetes/KubernetesLifecycle.java

```diff
@@ -1,36 +1,88 @@
 package org.opentosca.toscana.plugins.kubernetes;
 
 import java.io.IOException;
+import java.util.Set;
 
 import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.model.EffectiveModel;
+import org.opentosca.toscana.model.node.RootNode;
+import org.opentosca.toscana.plugins.kubernetes.exceptions.UnsupportedOsTypeException;
+import org.opentosca.toscana.plugins.kubernetes.visitor.NodeTypeCheckVisitor;
+import org.opentosca.toscana.plugins.kubernetes.visitor.OsCheckNodeVisitor;
 import org.opentosca.toscana.plugins.lifecycle.AbstractLifecycle;
 
+import org.slf4j.Logger;
+
 public class KubernetesLifecycle extends AbstractLifecycle {
+
+    private final EffectiveModel model;
+    private final Logger logger;
+
     /**
      @param context because the context is always needed this should never be null
      It probably gets called by the <code>getInstance</code> method of the LifecycleAwarePlugin
      */
     public KubernetesLifecycle(TransformationContext context) throws IOException {
         super(context);
+        model = context.getModel();
+        logger = context.getLogger(KubernetesLifecycle.class);
```

Comment:
You do not need to build a Logger in this context. The logger inherited form AbstractLifecycle is already in the proper Context (KubernetesLifecycle)